### DISCUSSION TITLE
Call for Presentations

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#87] [Silicon Salon]
We would love to get new presentations for a potential Silicon Salon II.

If you'd like to present on a topic related to semiconductor design for cryptography, whether it be as a manufacturer, user, developer, or designer, we want to hear your thoughts! Please, let us know that you'd like to present.

**Comment by kanzure** [comment]
I'd like to see a presentation from @stepansnigirev if possible.

He has previously given a related presentation:

1) https://diyhpl.us/wiki/transcripts/breaking-bitcoin/2019/future-of-hardware-wallets/

2) http://diyhpl.us/wiki/transcripts/austin-bitcoin-developers/2019-06-29-hardware-wallets/
